internal/server/v0/triple/golden/get_triples/observation.json

```diff
@@ -68,47 +68,6 @@
       "provenanceId": "dc/base/CensusPEP"
     }
   ],
-  "dc/o/em7gh269k7s2": [
```

Comment:
Observation is present in autopush https://screenshot.googleplex.com/BL8tdKj5pgK7Gob